arcgis-ios-sdk-samples/Maps/Map reference scale/MapReferenceScaleSettingsViewController.swift

```diff
@@ -0,0 +1,226 @@
+//
+// Copyright © 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+/// The delegate of a `MapReferenceScaleSettingsViewController`.
+protocol MapReferenceScaleSettingsViewControllerDelegate: AnyObject {
+    /// Tells the delegate that the user changed the map scale.
```

Comment:
Are these comment types (with three `///`) the recommended practice for Samples?

Reply:
Documentation comments (`/// …` or `/** … */`) are the recommended practice when documenting any symbol. That way the comments appear in Xcode's Quick Help:

![Screen Shot 2019-03-20 at 12 52 27 PM](https://user-images.githubusercontent.com/2257493/54714803-1e82ec00-4b0f-11e9-84ea-16881a8017dc.png)

I've followed that practice when writing comments above any symbol (with the exception of local variables) in all of the samples that I've implemented.

Reply:
Is that documented anywhere for the samples?  If more people are writing samples, it would be good to be consistent.

Reply:
Good question. I don't think it is in any sample-specific documentation, but we do have a backlog item for documenting our process. We could add this when working on that issue.